packages/i18n/en.json

```
@@ -257,7 +257,13 @@
       "title": "@:pages.admin.users",
       "addUser": "Add a user",
       "emptyResult": "No result found",
-      "result": "Showing {elements} out of {total} users."
+      "result": "Showing {elements} out of {total} users.",
+      "filters": {
+        "username": "Username",
+        "firstName": "FirstName",
+        "lastName": "LastName",
```

Prefer with a space ?

```suggestion
        "firstName": "First Name",
        "lastName": "Last Name",
```

---

My bad